.github/workflows/ci.yml

```diff
@@ -50,6 +50,7 @@ jobs:
           black . --check --diff
       - name: Type check with mypy
         run: |
+          mypy --install-types --non-interactive .
           mypy .
```

Comment:
first we run mypy just to see what types need to be installed, and install those types. Then we do the actual run